File: lesson_5_20240917/practice/GetInversion.java
package lesson_5_20240917.practice;

public class GetInversion {

    public static void main(String[] args) {
        int[] arr = /*{8, 4, 2, 1}*/ {3, 1, 2};
        int countInversion = getInversion(arr); 
        System.out.println("Count inversion = "+countInversion);
    }

    private static int getInversion(int[] arr) {
        int count = 0;
        for (int i = 0; i < arr.length; i++) {
            for (int j = i+1; j < arr.length; j++) {
                if(arr[i] > arr[j]) {
                    System.out.println(arr[i]+","+arr[j]);
                    count++;
                }
            }
        }
        return count;
    }

}

File: lesson_5_20240917/practice/get_inversion.js
function getInversion(arr) {
    let count = 0;
    for (let i = 0; i < arr.length; i++) {
        for (let j = i+1; j < arr.length; j++) {
            if(arr[i] > arr[j]) {
                console.log(arr[i]+","+arr[j]);
                count++;
            }
        }
    }
    return count;
}

let arr = [8, 4, 2, 1] /*[3, 1, 2]*/;
let countInversion = getInversion(arr); 
console.log("Count inversion = "+countInversion);